main.py

```diff
@@ -66,3 +69,68 @@ async def handle_events(self, event: KytosEvent):
                 TOPIC_NAME, event.name, event.name, event.content
             )
         )
+
+    @rest("v1/create", methods=['POST'])
```

Comment:
Usually a `create` in the URL with `POST` is a bit redundant it should ideally use the resource in its URL, since this doesn't have an openapi spec yet, it's even hard to understand. Also, `/v1/list` and `/v1/delete` `POST` aren't ideal either. Let's review the endpoints spec, and I recommend you also include an openapi.yml spec. Usually no verbs are included in the endpoints. You can consider using a `"v1/filter/"` or `"v1/event_filter"` or something along these lines, think about it.